dataland-csvconverter/uploadData.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Bitte in einer Readme oder im Wiki oder sonstwo den Prozess beschrieben, was man machen muss, wenn man ein neues Excel bekommt

Reply:
Im internen Wiki ist es beschrieben aber du meinst sicherlich auch im public Bereich, oder?

Reply:
das reicht mir da. Aber wo denn?

Reply:
Das hat eine eigene Seite im Wiki "Upload Company Data"

Reply:
ok